src/components/ec-modal/ec-modal.story.js

```diff
@@ -37,6 +37,9 @@ stories
       positiveHasText: {
         default: boolean('Positive Has Text', true),
       },
+      isLoading: {
+        default: object('Loading Positive Button', { positive: true, negative: false }),
+      },
```

Comment:
I'd split this into two booleans. It's easier to check/uncheck a checkbox than editing JSON object in textarea.

you can then change the template to:
`:isLoading="{ positive: isPositiveLoading, negative: isNegativeLoading }"`

Reply:
Good call!